### ISSUE TITLE
Questions about learning rate Update strategy

### ISSUE THREAD

**Issue by ksk2023** [ClaudiaCuttano/SAMWISE#24]
Dear Author,

Hello!The multi-GPU running instructions you provided earlier were of great help to me.However,I have some questions about the learning rate update strategy and would like to consult you.

You previously conducted training using the MeVis dataset,which contains 1,662 videos and 207,189 video segments.You originally set the learning rate to be updated once every 60,000 video segments.At that time, you were running on a single GPU,and the parameters were updated three times in one epoch(207,189/60,000=3.45315).

If I use four GPUs for running,then 3.45315/4<1,which means the learning rate will not change throughout the entire epoch.Do you have any suggestions for this issue?Or how can I improve the code?

Thank you for your help,and I look forward to your reply!

Best regards,

**Comment by hong-hd** [comment]
> 亲爱的作者，
> 
> 您好！您之前提供的多GPU运行指导对我帮助很大，但是我对学习率更新策略有一些疑问，想咨询您。
> 
> 你之前使用MeVis数据集进行训练，该数据集包含1,662个视频，207,189个视频片段。你最初设定的学习率是每60,000个视频片段更新一次。当时你在单GPU上运行，一个epoch内参数更新了三次（207,189/60,000=3.45315）。
> 
> 如果我使用四个 GPU 来运行，那么 3.45315/4<1，这意味着学习率在整个 epoch 中不会改变。您对这个问题有什么建议吗？或者我该如何改进代码？
> 
> 谢谢您的帮助，期待您的回复！
> 
> 此致，

Hi, have you solved it?

**Comment by ksk2023** [comment]
Hello! I  use the default setting about learning rate,I do not change the learning rate

**Comment by hong-hd** [comment]
> Hello! I use the default setting about learning rate,I do not change the learning rate

Thank you! I also use the default parameters, but I can't achieve the effect reported in the paper. I don't know where the problem is.

**Comment by ga1i13o** [comment]
Hi, realy sorry about the long delay.

The learning rate drop refers to the number of training steps. With a batch size of 4, an epoch on MeViS is slightly more thank 50k iterations so you would drop learning rate after the first epoch

**Comment by learncodingfromgit** [comment]
> > Hello! I use the default setting about learning rate,I do not change the learning rate
> 
> Thank you! I also use the default parameters, but I can't achieve the effect reported in the paper. I don't know where the problem is.

Hello, I’ve encountered the same issue as well and haven’t been able to achieve good experimental results. May I ask if you’ve resolved it?
